eta/core/datasets.py

```diff
@@ -444,6 +443,11 @@ def _iter_filtered_video_frames(video_dataset, frame_filter, stride):
                 yield frame_img, frame_labels, base_filename
 
 
+def _get_dataset_name(path):
+    base = os.path.basename(os.path.dirname(os.path.dirname(path)))
```

Comment:
I know this is private, but a docstrinig that just explains what `path` is would be nice